wikichron/dash/networks/models/CoEditingNetwork.py

```diff
@@ -42,13 +42,13 @@ class CoEditingNetwork(BaseNetwork):
     AVAILABLE_METRICS = {
             'Page Rank': 'page_rank',
             'Number of Edits': 'num_edits',
-            'Betweenness': 'betweenness'
+            'Betweenness': 'betweenness',
+            'Cluster': 'cluster'
         }
 
     USER_INFO = {
-        'Wiki ID': 'contributor_id',
+        'Wiki ID': 'id',
         'User Name': 'label',
-        'Edits Number': 'num_edits',
         'First Edit': 'first_edit',

```

Comment:
In my tests first_edit shows an integer number instead of the expected datetime like what last_edit does.

Reply:
Yep, I haven't changed it because I have to decide if I'll remove the First User and Last User attrs from the whole network, so then the string date will be easy to do for first_edit. So let it be for now :)